libdevcore/Common.h

```diff
@@ -332,13 +332,27 @@ int64_t utcTime();
 
 class ExitHandler {
 public:
+    enum exit_code_t {
+        ec_success = 0,
+        ec_failure = 1,  // same as EXIT_FAILURE in stdlib.h, generic failure in main()
+        ec_termninated_by_signal,
+        ec_compute_snapshot_error = 196,       // snapshot computation error
+        ec_rotation_complete = 197,            // exit requested after rotation complete
```

Comment:
Return code after completed node rotation should be 0, to avoid restarting of skaled container by docker